Makefile

```
@@ -178,10 +178,15 @@ gen-flat:
 
 ### Check vendored files and generated proto
 .PHONY: vendor-check
-vendor-check: gen-proto gen-flat
+vendor-check: gen-proto gen-flat update-mod
+	git diff --exit-code -- go.sum go.mod vendor/ pkg/tempopb/ pkg/tempofb/
```

I think we should include `cmd/tempo-serverless/go.sum` and `go.mod` in here as well. Otherwise they aren't included.

---

Or maybe we could do
```suggestion
	git diff --exit-code -- **/go.sum **/go.mod vendor/ pkg/tempopb/ pkg/tempofb/
```

---

Applied the suggestion. Nice catch, thanks